### PR TITLE
Add GHA config file for integration tests

### PR DESCRIPTION
**Pull request by elefeint**

### CODE REVIEW

**Review by dzou — APPROVED** [review]
Nice! Do you want to remove the Travis CI yaml + encrypted key in this PR too?

File: .github/workflows/integrationTests.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  schedule:
+    - cron: '00 8 * * *' # 08:00 UTC every day
+
+
+jobs:
+  integrationTests:
+    if: github.repository == 'GoogleCloudPlatform/cloud-spanner-r2dbc'
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
```
Comment (ttomsu) [inline .github/workflows/integrationTests.yaml:17]: You can remove the `strategy` block if not doing any matrix builds

**Review by ttomsu — APPROVED** [review]

**Comment by ttomsu** [comment]
You may need to just rerun the integration tests.

**Comment by elefeint** [comment]
It was a real problem that travis integration tests were masking! I had made this really questionable choice last year to avoid recreating tables by default. The problem is, that depending on the order in which the tests run, they need different column definitions. In the long run, these tests should either standardize the table schema required or use different tables. But for now, defaulting to recreating tables fixes the problem.

**Pull request merged**